### PR TITLE
Added some documentation about the gen_controller command

### PR DESCRIPTION
**Pull request by hectorj2f**
In relation to this comment: https://github.com/goadesign/goa/pull/1046#issuecomment-279288407

### CODE REVIEW

File: content/implement/goagen.ja.md [hunk 1]
```diff
@@ -1,108 +1,103 @@
 +++
```
Comment (raphael) [inline content/implement/goagen.ja.md]: Please don't override the Japanese version!

File: content/implement/goagen.md [hunk 2]
```diff
@@ -208,3 +193,16 @@ This command accepts one flag:
...
+## <a name="gen_controller"></a> Scaffolding: `goagen controller`
+
+The `controller` command generates a default (empty) implementation for each resource
+controller defined in the design package. By default this command only generates the files
+if they don't exist in the output directory. This command accepts the following flags:
+
+* `--force` causes the files to be generated even if files with the same name already exist.
+* `--res` specifies the name of the resource and is compulsory.
+* `--pkg` specifies the name of the generated controller package.
+* `--app-pkg` sets the new of the generated go package containing the controllers
```
Comment (raphael) [inline content/implement/goagen.md]: Typo `new` -> `name` also `go` -> `Go`
Maybe add "(defaults to `app`)"

**Comment by hectorj2f** [comment]
ping @raphael

**Comment by raphael** [comment]
Thanks!

**Pull request merged**